src/bin/lttng-sessiond/main.c

```diff
@@ -2904,6 +2908,115 @@ static unsigned int lttng_sessions_count(uid_t uid, gid_t gid)
 	return i;
 }
 
+static int receive_userspace_probe(struct command_ctx *cmd_ctx, int sock,
+			   int *sock_error, struct lttng_event *event)
+{
+	int fd, ret;
+	struct lttng_userspace_probe_location *probe_location;
+	struct lttng_userspace_probe_location_lookup_method *lookup = NULL;
+	struct lttng_dynamic_buffer probe_location_buffer;
+	struct lttng_buffer_view buffer_view;
+
+	/*
+	 * Create a buffer to store the serialized version of the probe
+	 * location.
+	 */
+	lttng_dynamic_buffer_init(&probe_location_buffer);
+	ret = lttng_dynamic_buffer_set_size( &probe_location_buffer,
+			cmd_ctx->lsm->u.enable.userspace_probe_location_len);
+	if (ret) {
+		ret = LTTNG_ERR_NOMEM;
+		goto error;
+	}
+
+	/*
+	 * Receive the probe location.
+	 */
+	ret = lttcomm_recv_unix_sock(sock, probe_location_buffer.data,
+			probe_location_buffer.size);
+	if (ret <= 0) {
+		DBG("Nothing recv() from client var len data... continuing");
+		*sock_error = 1;
+		lttng_dynamic_buffer_reset(&probe_location_buffer);
+		ret = LTTNG_ERR_PROBE_LOCATION_INVAL;
+		goto error;
+	}
+
+	buffer_view = lttng_buffer_view_from_dynamic_buffer(
+			&probe_location_buffer, 0, probe_location_buffer.size);
+
+	/*
+	 * Extract the probe location from the serialized version.
+	 */
+	ret = lttng_userspace_probe_location_create_from_buffer(
+				&buffer_view, &probe_location);
+	if (ret < 0) {
+		WARN("Failed to create a userspace probe location from the received buffer");
+		lttng_dynamic_buffer_reset( &probe_location_buffer);
+		ret = LTTNG_ERR_PROBE_LOCATION_INVAL;
+		goto error;
+	}
+
+	/*
+	 * Receive the file descriptor to the target binary from the client.
+	 */
+	DBG("Receiving userspace probe target FD from client ...");
+	ret = lttcomm_recv_fds_unix_sock(sock, &fd, 1);
+	if (ret <= 0) {
+		DBG("Nothing recv() from client userspace probe fd ... continuing");
```

Comment:
Remove space between `fd` and `...`

Reply:
It is to mimic the phrasing of the other stages (e.g.  var len filter) [1]

[1] https://github.com/frdeso/lttng-tools/blob/f227713ab853e5ad9340cd8cd233177bc33e7dfe/src/bin/lttng-sessiond/main.c#L3658